packages/pwa-kit-runtime/package.json

```diff
@@ -52,6 +52,7 @@
     "fs-extra": "^10.1.0",
     "internal-lib-build": "2.8.0-dev",
     "nock": "^13.3.0",
+    "pwa-kit-dev": "2.8.0-dev",
```

Comment:
pwa-kit-dev is already a dependency, is it still has to be listed as a peerDependency? 🤔

Reply:
Yes. It's been added as a _dev_ dependency. Peer dependencies apply to regular dependencies (for users of the package, rather than us maintainers).